### DISCUSSION TITLE
Multiselect using $oid

### DISCUSSION THREAD

**Discussion by 123srikan** [getredash/redash#6034] [General]
currently we use multiselect dropdown. my dropdown will have name and value.name will be text. value will be $oid.
i can do filter by single selection. but multiple selection by $oid is not working. can you please help me. please let me know if my question is clear

**Comment by 123srikan** [comment]
@redashsupport, can any one help me out?

**Comment by graciayashodara** [comment]
exactly what i encounter right now, have you solved this?
